addons/mail/static/src/models/thread/thread.js

```diff
@@ -1126,6 +1174,22 @@ function factory(dependencies) {
             return [['unlink']];
         }
 
+        /**
+         * @private
+         * @returns {mail.message|undefined}
+         */
+        _computeLastTransientMessage() {
+            if (this.orderedMessages.length === 0) {
+                return [['unlink']];
+            }
+            for (let i = this.orderedMessages.length - 1; i >= 0; i--) {
+                const message = this.orderedMessages[i];
+                if (message.isTransient) {
+                    return [['link', message]];
+                }
+            }
```

Comment:
Note: if we use this iteration only once then fine. But if/when used multiple times we should consider adding a field `orderedTransientMessages` to only trigger all of those loops once if there is actually a change of transient message, instead of redoing this on any message change.

Reply:
Not sure what's best as this loop would be needed to recompute orderedTransientMessages when orderedMessages changes if we do it that way, and here it will be triggered when orderedMessages changes too .. so I'm not sure there is a difference ?

Reply:
Indeed for now there would be no difference, that's why it was just a note. That would make a difference only if we used "orderedTransientMessages" (currently equivalente this loop) in multiple places, which is currently not the case.

Reply:
Would you say we do the change now or when the described situation occurs ?

Reply:
When it occurs is fine, we can resolve this for now.